src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts

```diff
@@ -27,7 +27,7 @@ export class SellerBidListComponent implements OnInit {
     this.seller = this.appServ.allSellers.filter(seller => seller.id == this.sellerId)[0];
 
     this.appServ.getAllBids().subscribe(data => {
-      this.bids = data.filter(bid => bid.sellerId == +this.sellerId);
+      this.bids = data.filter(bid => bid.sellerId == +this.sellerId).sort((a, b) => b.id - a.id)
```

Comment:
@subbu3012 - same as the previous: sorting in the frontend for paginated data from the backend will not work.

Reply:
@vraravam please refer to the above comment. https://github.com/chennaitricolor/wastexchange-fe/pull/41#discussion_r358742292

Reply:
Got it